travis-php.ini

```diff
@@ -0,0 +1 @@
+extension = "mongo.so"
```

Comment:
Missing new line. Please adhere the settings in `.editorconfig` file. Check if there is a plugin for your IDE: http://editorconfig.org/